src/corecel/sys/ScopedDeviceProfiling.cuda.cc

```diff
@@ -20,24 +20,38 @@
 
 namespace celeritas
 {
+namespace
+{
+nvtxDomainHandle_t domain_handle()
+{
+    static nvtxDomainHandle_t domain = nvtxDomainCreateA("Celeritas");
+    return domain;
+}
+nvtxEventAttributes_t attributes(char const* name)
```

Comment:
```suggestion
nvtxEventAttributes_t make_attributes(char const* name)
```
since it's returning a new object by value as opposed to accessing something